examples/ARC_I2C1/master_writer/master_writer.ino

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2016 Intel Corporation.  All rights reserved.
+// See the bottom of this file for the license terms.
+
+/*
+ *  This sketch is to demonstrate I2C writing to a device, as a bus master.  It first makes a request to
+ *  write to the device.  Then proceed with write out one byte annd completed the transmission.  The sketch
+ *  continues with these steps and increament the byte value each lap.  If the write operation fails, please check
+ *  the connection to the external device:  verify the signals, make sure the board and the device share a
+ *  common ground connection.
+ *
+ *  Please note:
+ *    1. This sketch makes use of ARC_I2C1.  One of the two I2C connections from the ARC core.  For the QUARK core I2C
+ *       connection, please refer to the quarkI2C library.
+ *    2. The ARC_I2C1 connections are not available in the Arduino 101 hardware platform.
+ *    3. The sketch is accessing an external I2C device with an device number (address) of 8.  This address can be
+ *       changed to fit your setup.
+ */
+
+
+#include <ARC_I2C1.h>
+
+void setup()
+{
+  Serial.begin(9600);   // start serial for output
+  while(!Serial);       // Wait here till serial terminal is ready/opened
+
+  ARC_I2C1.begin(); // join i2c bus
+}
+
+byte x = 1;
```

Comment:
one-letter variable names in sketches, not great for the user. How about something more descriptive, like `sendData` and `receiveData`